tests/test_federal_senate_dataset.py

```diff
@@ -1,41 +1,92 @@
-import os
 from tempfile import gettempdir
-from unittest import main, skipIf, TestCase, TestLoader
+from unittest import TestCase
+from unittest.mock import patch
+import pandas as pd
+import os
```

Comment:
It is very important to pay attention to the imports order and separating lines between types of imports, this imports should be:
```
import os
from tempfile import gettempdir
from unittest import TestCase
from unittest.mock import patch

import pandas as pd
```
Refer to [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) for further instructions.

Particularly right now I'm using [isort](https://pypi.python.org/pypi/isort) to organize my imports, I believe @cuducos has other software he could recommend (can't remember the name)